### DISCUSSION TITLE
IDAES sample program （subcritical_power_plant）

### DISCUSSION THREAD

**Discussion by weilongfei123** [IDAES/idaes-pse#1313] [Q&A]
This program is in (IDAES/models _ extra/power _ generation/flowsheets/subcritical _ power _  plant).
        

1. When running the dynamic model of the whole model, the degree of freedom of the model is not 0.
2. Parameter "m_dyn.fs_main.fs_stc.spray_ctrl.mv_integral_component_ref[t0]" and parameter "m_dyn.fs_main.fs_stc.makeup_ctrl.mv_integral_component_ref[t0]"（lines 1875 to1880） are missing after forcibly changing the degree of freedom to 0.
3. After changing the missing parameters to constants, the model runs slowly and the results are wrong.

**Comment by weilongfei123** [comment]
This is the result of running the program after my own modification. This result is wrong. There is almost no relationship between power demand and actual power output.
![16388aacc1e3b2e8fa7150898fdd82df](https://github.com/IDAES/idaes-pse/assets/102841533/fc342e18-3196-4b66-8f05-95821a79267f)

**Reply by weilongfei123** [reply]
Please help me from all walks of life.

**Comment by Peng-debug-del** [comment]
Hello, have you solved this problem? If it has been resolved, could you please share how you did it? I have the same issue as you.
